spec/swagger.yaml

```
@@ -1,69 +1,66 @@
 swagger: '2.0'
 info:
   version: 2.0.1
-  title: RBKmoney Platform API
+  title: RBK.money Platform API
   description: >
+    #### Последнее обновление: 14-09-2021 | 
+    
     ## Описание
 
-    RBKmoney API является базовой и единственной точкой взаимодействия с
-    платежной платформой. Все изменения состояний платформы осуществляются с
-    помощью вызовов соответствующих методов API. Любые сторонние приложения,
-    включая наши веб-сайты и другие UI-интерфейсы являются внешними
-    приложениями-клиентами.
+    RBK.money API предназначен для [мерчантов](https://developer.rbk.money/docs/payments/overview/#lk), принимающих
+    платежи из своего пользовательского интерфейса (веб-сайта или любого другого),
+    и является единственной точкой взаимодействия с [платформой](https://developer.rbk.money/docs/payments/overview/) для проведения операций оплаты товаров/услуг.
 
-    RBKmoney API работает поверх HTTP-протокола. Мы используем REST архитектуру,
-    схема описывается в соответствии со [Swagger](http://swagger.io/). Коды
-    возврата описываются соответствующими HTTP-статусами. Платформа принимает и
-    возвращает JSON-структуры в HTTP body.
+    ## Сценарии
 
-    ## Запросы
+    Примеры информационного взаимодействия для некоторых пользовательских сценариев представлены в данных статьях:
 
-    Любой вызов методов API обязан предваряться предоставлением уникального для
-    участника в пределах платформы ID запроса. Данный ID передается в
-    соответствующем заголовке HTTP-запроса:
+      - оплата [заказа](https://developer.rbk.money/docs/payments/overview/#invoice) покупателем: см. ["Платеж"](https://developer.rbk.money/docs/payments/overview/#pay);
+      - [рекуррентный платёж](https://developer.rbk.money/docs/payments/recurring/recurring/#_1);
+      - [покупка с привязанной карты](https://developer.rbk.money/docs/payments/examples/binding/).
+    
+    ## Детали взаимодействия
+
+    При любом обращении к API в заголовке соответствующего запроса необходимо передать его уникальный идентификатор:
 
     ```
      X-Request-ID: oX5MWM2AQy
     ```
 
-    ## Тип содержимого и кодировка
+    ### Тип содержимого и кодировка
 
-    Любой запрос к API должен выполняться в кодировке UTF-8 и с указанием
-    содержимого в формате JSON
+    [Платформа](https://developer.rbk.money/docs/payments/overview/) принимает и возвращает данные в формате JSON и кодировке UTF-8:
 
     ```
       Content-Type: application/json; charset=utf-8
     ```
 
-    ## Формат дат
+    ### Формат дат
 
-    Платформа принимает значения date-time в формате, описанном в RFC 3339, например:
+    [Платформа](https://developer.rbk.money/docs/payments/overview/) принимает и возвращает значения date-time в формате, описанном в [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339):
```

Быть может тогда как-то так?
```suggestion
    [Платформа](https://developer.rbk.money/docs/payments/overview/) принимает и возвращает значения отметок во времени в формате `date-time`, описанном в [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339):
```

---

Может так?

[Платформа](https://developer.rbk.money/docs/payments/overview/) принимает и возвращает значение отметки времени в формате `date-time`, описанном в [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339):

---

Нет особых возражений, кроме разве что единственного числа.

---

с чем связано возражение про единственное число?

---

С тем, что это воспринимается так, словно мы возвращаем _только_ отметку времени и больше ничего.

---

ок

[Платформа](https://developer.rbk.money/docs/payments/overview/) принимает и возвращает значение отметок времени в формате `date-time`, описанном в [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339):